packages/table/src/table.tsx

```diff
@@ -425,7 +427,10 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
     // when true, we'll need to imperatively synchronize quadrant views after
     // the update. this variable lets us avoid expensively diff'ing columnWidths
     // and rowHeights in <TableQuadrantStack> on each update.
-    private didUpdateColumnOrRowSizes: boolean = false;
+    private didUpdateColumnOrRowSizes = false;
+
+    // when true, indicates that all cells have been completely mounted.
```

Comment:
Is this comment helpful? Also, I'm going to rename this `hasCompletelyRendered` given that `onCompleteRender` is currently invoked both on mount and on update.